phalcon/acl/roleable.zep

```diff
@@ -0,0 +1,33 @@
+
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2015 Phalcon Team (http://www.phalconphp.com)       |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file docs/LICENSE.txt.                        |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ +------------------------------------------------------------------------+
+*/
+
+namespace Phalcon\Acl;
+
+/**
+ * Phalcon\Acl\Roleable
+ *
+ * Interface for classes which could be used in allow method as ROLE, RoleInterace need __construct method so i decided to do this simple interface
+ */
+interface Roleable
+{
+    /**
+     * Returns the role name
+    */
+    public function getRoleName() -> string;
```

Comment:
https://github.com/phalcon/cphalcon/blob/master/phalcon/acl/roleinterface.zep#L38 is not enough?